x-pack/plugins/apm/common/__snapshots__/constants.test.ts.snap

```diff
@@ -194,6 +230,18 @@ exports[`Transaction v2 ERROR_LOG_MESSAGE 1`] = `undefined`;
 
 exports[`Transaction v2 ERROR_LOG_STACKTRACE 1`] = `undefined`;
 
+exports[`Transaction v2 METRIC_PROCESS_CPU_PERCENT 1`] = `undefined`;
+
+exports[`Transaction v2 METRIC_PROCESS_MEMORY_RSS 1`] = `undefined`;
+
+exports[`Transaction v2 METRIC_PROCESS_MEMORY_SIZE 1`] = `undefined`;
+
+exports[`Transaction v2 METRIC_SYSTEM_CPU_PERCENT 1`] = `undefined`;
+
+exports[`Transaction v2 METRIC_SYSTEM_FREE_MEMORY 1`] = `undefined`;
+
+exports[`Transaction v2 METRIC_SYSTEM_TOTAL_MEMORY 1`] = `undefined`;
+
```

Comment:
Seems like we need a better solution here. I'll be adding two additional sections for error constants also (v1, and v2), and that'll duplicate the metric constants twice more.
Anything in mind?

Reply:
I almost tried to solve it and then backed away haha, but yeah I'm with you. We probably need to test groups of config against mock documents that have the expected key paths ... we are doing it only against transactions and spans currently. I wasn't sure if we wanted a "Metric" type for the document or just an ad hoc mock metric doc... etc